doc/03-Configuration.md

```diff
@@ -23,7 +23,7 @@ Data resource where IcingaDB stores synced data and historical events.
 
 Option                   | Description
 -------------------------|-----------------------------------------------
-host                     | **Optional.** MySQL host. Defaults to `127.0.0.1`.
+host                     | **Optional.** MySQL host. Defaults to `127.0.0.1`. It is also possible to use unix socket here by entering the absolute path to ``mysql.sock``.
```

Comment:
Combine and shorten that:

"MySQL host." => "MySQL host or absolute Unix socket path."